test/integration/resourcemanager/health/health_suite_test.go

```diff
@@ -20,6 +20,8 @@ import (
 	"testing"
 	"time"
 
+	"k8s.io/client-go/rest"
```

Comment:
Nit: could be moved to the last import group. There are also other occurrences, I would appreciate if you can tackle them as well, I won't comment about them to avoid spamming the same comment.